src/main/java/snomui/Ui.java

```diff
@@ -0,0 +1,44 @@
+package snomui;
+
+import java.util.Scanner;
+
+public class Ui {
+    private Scanner input;
```

Comment:
perhaps use other names rather than`input` as Scanner Class is input device, not input 🤔